runtime/test/manifest-test.js

```diff
@@ -470,8 +470,8 @@ describe('manifest', function() {
         }
         ++count;
       }
-    })
-    assert.equal(count, 17);
+    });
+    assert.equal(count, 18);
```

Comment:
Is this why travis is failing, maybe a bad merge or part of a different change?

Reply:
Yeah, i think this is a result of my "get rid of interleaved hack" pr that Doug just recently merged

Reply:
Oh, oops! I messed up the rebase.